src/config_generator/openapi_spec/jsonplaceholder.yml

```diff
@@ -0,0 +1,66 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: JSON Placeholder API
+  description: See https://jsonplaceholder.typicode.com/
+servers:
+  - url: http://jsonplaceholder.typicode.com
+paths:
+  /posts:
+    get:
+      description: Returns all posts
+      tags: ["Posts"]
+      operationId: "getPosts"
+      responses:
+        "200":
+          description: Successful response
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/PostsList"
+
+  /posts/{id}:
+    get:
+      description: Returns a post by id
+      tags: ["Posts"]
+      operationId: "getPost"
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: The user id.
+          schema:
+            type: integer
+            format: int64
+      responses:
+        "200":
+          description: Successful response
+          content:
+            "application/json":
+              schema:
+                $ref: "#/components/schemas/Post"
+        "404":
+          description: Post not found
+
+components:
+  schemas:
+    PostsList:
+      "type": "array"
+      "items":
+        $ref: "#/components/schemas/Post"
+    Post:
+      "type": "object"
+      "required":
+        - "id"
+        - "userId"
+        - "title"
+        - "completed"
+      "properties":
+        id:
+          type: "integer"
+        userId:
+          type: "integer"
+        title:
+          type: "string"
+        completed:
+          type: "string"
```

Comment:
Download and add tests for more complex open API specifications.

Reply:
Stripe, Spotify, Github more over here https://github.com/public-apis/public-apis?tab=readme-ov-file